### PR TITLE
separate yaml file for specific interfaces

### PR DESCRIPTION
**Pull request by smruti77**
three yaml file for vnic,sriov,veth with supported option.

Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

**Comment by smruti77** [comment]
JOB ID     : 2ca7a502913592a81ef10424f4a69ff24dde8852
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-07-17T09.39-2ca7a50/job.log
 (1/7) ethtool_test.py:Ethtool.test_ethtool;arg-b5ac: FAIL: ping failed after interface is up (15.22 s)
 (2/7) ethtool_test.py:Ethtool.test_ethtool;Driver-7bd5: FAIL: ping failed after interface is up (15.15 s)
 (3/7) ethtool_test.py:Ethtool.test_ethtool;RingParameter-0c2f: FAIL: failed (0.92 s)
 (4/7) ethtool_test.py:Ethtool.test_ethtool;ShowOffloadProtocolFeature-cf93: FAIL: ping failed after interface is up (15.17 s)
 (5/7) ethtool_test.py:Ethtool.test_ethtool;Statistics-ac99: FAIL: ping failed after interface is up (15.24 s)
 (6/7) ethtool_test.py:Ethtool.test_ethtool;Show-permaddr-95b3: FAIL: ping failed after interface is up (15.12 s)
 (7/7) ethtool_test.py:Ethtool.test_ethtool;Show-time-stamping-c6ab: FAIL: ping failed after interface is up (15.16 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 7 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 93.02 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-07-17T09.39-2ca7a50/results.html
ltcfleet2-lp9:~/avocado-misc-tests/io/net # avocado run ethtool_test.py -m ethtool_test.py.data/ethtool_veth_test.yaml
JOB ID     : 7bffa339ef08d22e0f8c242f2d75c941c6af4025
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-07-17T09.42-7bffa33/job.log
 (1/6) ethtool_test.py:Ethtool.test_ethtool;arg-ead4: PASS (5.35 s)
 (2/6) ethtool_test.py:Ethtool.test_ethtool;Driver-f61d: PASS (5.33 s)
 (3/6) ethtool_test.py:Ethtool.test_ethtool;ShowOffloadProtocolFeature-68a9: PASS (5.34 s)
 (4/6) ethtool_test.py:Ethtool.test_ethtool;Statistics-8f64: PASS (5.33 s)
 (5/6) ethtool_test.py:Ethtool.test_ethtool;Show-permaddr-ab15: PASS (5.33 s)
 (6/6) ethtool_test.py:Ethtool.test_ethtool;Show-time-stamping-0576: PASS (5.34 s)
RESULTS    : PASS 6 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 32.85 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-07-17T09.42-7bffa33/results.html
ltcfleet2-lp9:~/avocado-misc-tests/io/net # avocado run ethtool_test.py -m ethtool_test.py.data/ethtool_sriov_test.yaml
JOB ID     : 604087c3f1f84261060d2c993e87cd55fe34e935
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-07-17T09.49-604087c/job.log
 (1/9) ethtool_test.py:Ethtool.test_ethtool;arg-aab7: FAIL: ping failed after interface is up (14.70 s)
 (2/9) ethtool_test.py:Ethtool.test_ethtool;Driver-7bd9: FAIL: ping failed after interface is up (14.70 s)
 (3/9) ethtool_test.py:Ethtool.test_ethtool;PauseParameter-1189: FAIL: ping failed after interface is up (14.59 s)
 (4/9) ethtool_test.py:Ethtool.test_ethtool;Coalescing-2514: FAIL: ping failed after interface is up (14.61 s)
 (5/9) ethtool_test.py:Ethtool.test_ethtool;RingParameter-95fe: FAIL: ping failed after interface is up (14.60 s)
 (6/9) ethtool_test.py:Ethtool.test_ethtool;ShowOffloadProtocolFeature-2e52: FAIL: ping failed after interface is up (14.67 s)
 (7/9) ethtool_test.py:Ethtool.test_ethtool;Statistics-80ff: FAIL: ping failed after interface is up (14.75 s)
 (8/9) ethtool_test.py:Ethtool.test_ethtool;Show-permaddr-35aa: FAIL: ping failed after interface is up (14.66 s)
 (9/9) ethtool_test.py:Ethtool.test_ethtool;Show-time-stamping-bd43: FAIL: ping failed after interface is up (14.60 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 9 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 133.12 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-07-17T09.49-604087c/results.html

File: io/net/ethtool_test.py.data/ethtool_srivo_test.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+nterface:
```
Comment (narasimhan-v) [inline io/net/ethtool_test.py.data/ethtool_srivo_test.yaml]: interface

File: io/net/ethtool_test.py.data/ethtool_veth_test.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+nterface:
```
Comment (narasimhan-v) [inline io/net/ethtool_test.py.data/ethtool_veth_test.yaml]: interface

File: io/net/ethtool_test.py.data/ethtool_vnic_test.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+nterface:
```
Comment (narasimhan-v) [inline io/net/ethtool_test.py.data/ethtool_vnic_test.yaml]: interface

**Pull request closed without merging**